src/deploy/functions/deploy.ts

```diff
@@ -4,13 +4,11 @@ import { setGracefulCleanup } from "tmp";
 import { checkHttpIam } from "./checkIam";
 import { functionsUploadRegion } from "../../api";
 import { logSuccess, logWarning } from "../../utils";
+import { Options } from "../../options";
 import * as args from "./args";
-import * as backend from "./backend";
 import * as fs from "fs";
```

Comment:
Not your change, but while you're here - move fs to the external imports block

Reply:
Done